From: Mario Limonciello <supermario@mlimonciello.(none)>
Date: Fri, 4 Apr 2008 20:51:26 +0000 (-0500)
Subject: UBUNTU: SAUCE: Enable Reset and SCO workaround on Dell 410 BT adapter
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=7dcc5660eeba55644c4c5775097fb3f2d91d1602

UBUNTU: SAUCE: Enable Reset and SCO workaround on Dell 410 BT adapter

OriginalAuthor: Mario Limonciello
Ignore: no

When the Dell 410 BT adapter is put into BT radio mode, it needs
to be prepared like many other Broadcom adapters.

Signed-off-by: Mario Limonciello <Mario_Limonciello@Dell.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---

Index: 2008-devel/drivers/bluetooth/hci_usb.c
===================================================================
--- 2008-devel.orig/drivers/bluetooth/hci_usb.c
+++ 2008-devel/drivers/bluetooth/hci_usb.c
@@ -136,6 +136,8 @@ static struct usb_device_id blacklist_id
 	{ USB_DEVICE(0x413c, 0x8126), .driver_info = HCI_RESET | HCI_WRONG_SCO_MTU },
 	/* Dell Wireless 370 */
 	{ USB_DEVICE(0x413c, 0x8156), .driver_info = HCI_RESET | HCI_WRONG_SCO_MTU },
+	/* Dell Wireless 410 */
+	{ USB_DEVICE(0x413c, 0x8152), .driver_info = HCI_RESET | HCI_WRONG_SCO_MTU },
 
 	/* Microsoft Wireless Transceiver for Bluetooth 2.0 */
 	{ USB_DEVICE(0x045e, 0x009c), .driver_info = HCI_RESET },
